tests/gold_tests/pluginTest/lua/lua_states_stats.test.py

```diff
@@ -28,12 +28,7 @@
 # Define default ATS
 server = Test.MakeOriginServer("server")
 
-# It is necessary to redirect stderr to a file so it will be available for examination by a test run.
-ts = Test.MakeATSProcess(
-    "ts", command="traffic_manager 2> " + Test.RunDirectory + "/ts.stderr.txt", select_ports=True
-)
```

Comment:
This is an example of something no longer necessary if we bind stdout and stderr to traffic.out. Instead we can simply interact with traffic.out.